src/common/ceph_time.cc

```diff
@@ -95,7 +95,7 @@ namespace ceph {
 	   typename std::enable_if<Clock::is_steady>::type*>
   std::ostream& operator<<(std::ostream& m,
 			   const std::chrono::time_point<Clock>& t) {
-    return m << std::chrono::duration<double>(t.time_since_epoch()).count()
+    return m << std::chrono::duration_cast<seconds>(t.time_since_epoch()).count()
```

Comment:
@adamemerson PTAL, this looks good?

Reply:
I don't know if we really want to disable printing subseconds. I'd rather either use iomanip to specify the floating point format and disallow scientific notation that way.

(Or just skip the conversion to double and use fmt/chrono.h.)

Reply:
I have modified the code to use iomanip.